source/MaterialXView/Viewer.cpp

```diff
@@ -2467,3 +2467,28 @@ void Viewer::updateAlbedoTable()
     glBindFramebuffer(GL_DRAW_FRAMEBUFFER, 0);
     glDrawBuffer(GL_BACK);
 }
+
+void Viewer::renderScreenSpaceQuad(MaterialPtr material)
+{
+    mx::MeshStreamPtr quadPositions = mx::MeshStream::create(mx::HW::IN_POSITION, mx::MeshStream::POSITION_ATTRIBUTE, 0);
```

Comment:
I'd suggest we do this small short term things: 

1. Add a  "quad creation" utility in GeometryHandler.h. As discussed this should be stream requirement driven based on the material though this works for now for shaders which only require position + placement so can leave as is.
2. Cache this mesh on the Viewer and create it only once.

I think ideally this should be "stock" geometry actually cached in the GeometryHandler but can go with this for now.

Reply:
If / when it get's migrated then we should be able to replace drawScreenSpaceQuad() -- but for later :).

Reply:
I'll add a utility in GeometryHandler